### PR TITLE
docs: Add a doc requirements doc

### LINKED ISSUE #99: Add documentation requirements document
Add a document that specifies some basic requirements for project documentation.

Replaces #91, #94.

### PR DESCRIPTION
**Pull request by jodh-intel**
Add a document that explains the expectations for project
documentation.

Fixes #99.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Hi @egernst, @klynnrif - ptal at this which replaces #94 (with https://github.com/kata-containers/tests/pull/316 too).

File: Documentation-Requirements.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+- All documents are expected to be written in GitHub Flavored Markdown format.
+- All documents should have the `.md` file extension.
+
+# Code blocks
+
+This section lists requirements for displaying commands and command output.
+
+The requirements must be adhered to since documentation containing code blocks
+is validated by the CI system which executes the command blocks with the help
```
Comment (klynnrif) [inline Documentation-Requirements.md]: missing a comma after "system": is validated by the CI system, which executes the command blocks with the help

Comment (jodh-intel) [reply Documentation-Requirements.md]: So close! :smile: Branch updated.

File: Documentation-Requirements.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* [Code blocks](#code-blocks)
+
+# Introduction
+
+This document outlines the requirements for all documentation in the [Kata
+Containers](https://github.com/kata-containers) project.
+
+# General requirements
+
+- All documents are expected to be written in GitHub Flavored Markdown format.
```
Comment (grahamwhaley) [inline Documentation-Requirements.md]: You could hotlink to [Github Flavored Markdown](https://github.github.com/gfm/)

Comment (jodh-intel) [reply Documentation-Requirements.md]: Good point - done.

**Pull request merged**